extensions/amp-story/1.0/animation.js

```diff
@@ -192,9 +204,20 @@ export class AnimationRunner {
    */
   getDims() {
     return this.vsync_.measurePromise(() => {
-      const targetRect = unscaledClientRect(this.target_);
+      const target = dev().assertElement(this.presetTarget_);
+      const targetRect = unscaledClientRect(target);
       const pageRect = unscaledClientRect(this.page_);
 
+      // TODO(alanorozco): Expose equivalents to <amp-animation>
+      // - targetWidth/targetHeight are already available as width()/height()
+      // - pageWidth/pageHeight should be exposed as vw/vh
+      // - targetX/targetY should be exposed somehow (?)
+      //
+      // TODO(alanorozco): After exposing these to <amp-animation> syntax, we
```

Comment:
Nit: please add the issue ID here and above

Reply:
This was already added but Github was showing an older set of commits by default (???)